apps/shipments/filters.py

```diff
@@ -19,3 +19,19 @@ class Meta:
                   'final_destination_location__state', 'final_destination_location__address_1', 'ship_to_location',
                   'final_destination_location__postal_code', 'final_destination_location__country',
                   'final_destination_location', 'ship_from_location__state',]
+
+
+class ShipmentHistoryFilter(ShipmentFilter):
+    date_lesser = filters.DateFilter(field_name="updated_at", lookup_expr='lte')
+    date_greater = filters.DateFilter(field_name="updated_at", lookup_expr='gte')
+
+    class Meta:
+        model = Shipment.history.model
+        fields = ['mode_of_transport_code', 'ship_from_location__name', 'ship_from_location__city',
```

Comment:
Once we move this to be scoped under `/api/v1/shipments/{id}/`, I don't think we will need any shipment field filters.